src/document-register-element.js

```diff
@@ -607,10 +607,10 @@ document[REGISTER_ELEMENT] = function registerElement(type, options) {
       return node;
     };
 
-    HTMLElementPrototype.cloneNode = function (deep) {
+    HTMLElementPrototype.cloneNode = DocumentFragment.prototype.cloneNode = function (deep) {
       var
         node = cloneNode.call(this, !!deep),
```

Comment:
also this line uses `cloneNode` from `HTMLElement.prototype` which is not the one from `DocumentFragment.prototype` so this might actually, and easily, fail